src/GraphQL.NewtonsoftJson/GraphQLContractResolver.cs

```diff
@@ -21,6 +23,24 @@ protected override JsonConverter ResolveContractConverter(Type objectType)
             if (typeof(ExecutionResult).IsAssignableFrom(objectType))
                 return new ExecutionResultJsonConverter(_errorInfoProvider, NamingStrategy);
 
+            if (typeof(Inputs) == objectType)
```

Comment:
https://en.wikipedia.org/wiki/Yoda_conditions

```suggestion
            if (objectType == typeof(Inputs))
```
here and one more below